packages/mobile/src/components/Chat/Chat.component.tsx

```diff
@@ -126,3 +132,21 @@ const customInputStyle = StyleSheet.create({
     borderWidth: 0
   }
 })
+
+export const ChannelMessagesComponent: React.FC<ChannelMessagesComponentProps> = ({ messages }) => {
+  return (
+    <>
+      {Object.keys(messages).map(day => {
+        return (
+          <View key={day}>
+            {/* <MessagesDivider title={day} /> */}
```

Comment:
Do we need that comment here?

Reply:
I think so. Just to remind us that we should place MessagesDivider component (as soon as we add it) in this exact place

Reply:
I needed to adapt this component so we're able to display messages without having to change the nectars' selectors